### DISCUSSION TITLE
error Malformed UTF-8 characters, possibly incorrectly encoded in update

### DISCUSSION THREAD

**Discussion by ferasat** [livewire/livewire#7540] [1. Help]
I have the "Malformed UTF-8 characters, possibly incorrectly encoded in update" error in the code below When I call changStatus().

in blade : 
`
@foreach($posts as $post)
                    @if($statusShowPosts == 0)
                        <div class="card mb-3">
                            <img src="{{ asset($post->pic) }}" class="card-img card-img-top" alt="{{ $post->name }}">
                            <div class="card-body">
                                <h4 class="card-title mb-3">
                                    <a href="{{ asset('post/'.$post->slug) }}" class="stretched-link">
                            <span class="overflow-hidden">
                                {{ $post->name }}
                            </span>
                                    </a>
                                </h4>
                                <div class="card-text">
                                    {!! text_summary($post->texts) !!}
                                </div>
                            </div>
                            <div class="card-footer">
                                <a class="btn btn-outline-primary" href="{{ asset('post/'.$post->slug) }}">اطلاعات
                                    بیشتر</a>
                            </div>
                        </div>
                    @elseif($statusShowPosts == 1)
                        5555
                        <div class="col-md-4">
                            <div class="cart">
                                <img src="{{ asset($post->pic) }}" class="card-img card-img-top"
                                     alt="{{ $post->name }}">
                                <div class="card-body">
                                    <h4 class="card-title mb-3">
                                        <a href="{{ asset('post/'.$post->slug) }}" class="stretched-link">
                                        <span class="overflow-hidden">
                                            {{ $post->name }}
                                        </span>
                                        </a>
                                    </h4>
                                    <div class="card-text">
                                        {!! text_summary($post->texts) !!}
                                    </div>
                                </div>
                                <div class="card-footer">
                                    <a class="btn btn-outline-primary" href="{{ asset('post/'.$post->slug) }}">اطلاعات
                                        بیشتر</a>
                                </div>
                            </div>
                        </div>
                    @endif
                @endforeach
`

in class:
`
use WithPagination;

    protected $paginationTheme = 'bootstrap';
    public $show = false, $statusShowPosts =0, $i = 1, $countSlide = 0;
    public $statusBlog , $cat_id ;

    public function render()
    {
        if ($this->statusBlog == 'cat'){
            $posts = Post::where([
                'typePost' => 'post',
                'statusPublish' => 'publish',
                'cat_id' => $this->cat_id
            ])->orderBy('id', 'desc')->paginate(18);
        }else{
            $posts = Post::where([
                'typePost' => 'post',
                'statusPublish' => 'publish'
            ])->orderBy('id', 'desc')->paginate(18);
        }

        $post3 = Post::where([
            'typePost' => 'post',
            'statusPublish' => 'publish',
            'specialPin' => 1
        ])->orderBy('id', 'desc')->take(3)->get();

        $dl_drivers = Post::query()->where('statusPublish' , 'publish',)->where('name', 'LIKE', '%' . 'دانلود درایور' . "%")->get();

        $this->countSlide = count($post3);

        return view('livewire.user.post.blog.blog', compact('posts', 'post3' , 'dl_drivers'));
    }

    public function changStatus()
    {
        //dd(2222);
        if ($this->statusShowPosts == 0)
            $this->statusShowPosts = 1;
        else $this->statusShowPosts = 0;
        $this->render();
    }
`

**Comment by lucenarenato** [comment]
Solution?
